src/Microsoft.ML.Transforms/MissingValueReplacing.cs

```diff
@@ -896,16 +896,14 @@ public void SaveAsOnnx(OnnxContext ctx)
 
             private bool SaveAsOnnxCore(OnnxContext ctx, int iinfo, ColInfo info, string srcVariableName, string dstVariableName)
             {
-                DataKind rawKind;
+                Type rawType;
                 var type = _infos[iinfo].TypeSrc;
                 if (type is VectorType vectorType)
-                    rawKind = vectorType.ItemType.RawKind;
-                else if (type is KeyType keyType)
-                    rawKind = keyType.RawKind;
+                    rawType = vectorType.ItemType.RawType;
```

Comment:
>rawType = vectorType.ItemType.RawType; [](start = 20, length = 38)

Placeholder note, once your other PR goes in you could probably benefit from your new `GetItemType` extension.